_posts/2020-04-07-charity-shop-exchange.md

```diff
@@ -0,0 +1,63 @@
+
+Our product owner, Kelvin Lockwood, conceptualized the idea in response to the challenges we face as a world now under quarantine. Older or immunocompromised people cannot leave their homes, but are in need of ways to stay engaged. In the UK there are over 10,000 charity shops still accepting donated goods, and a population of 66 million, many of whom are facing extended lockdowns. 
+
+**Can you describe your project?**
+
+Charity Shop Exchange offers a subscription service of books & DVDs. People need new things to read and watch - especially when feeling isolated - and our subscription service facilitates this in an economical way. Subscribers receive a (weekly, bi-weekly or monthly)  selection of books and DVDs which have been sourced from local charity shops.
+
+**How did your team come together?**
+
+All of us met through *Slack*, or *Twitter* while trying to discover a way we could help out during this crisis. Kelvin had taken the steps to reach out to us individually, add us to a Slack channel so we could discuss the concept, and from there we got right down to it!
+
+**Can you tell us about your team?**
+
+*Kelvin Lockwood* hails from Bournemouth, UK  
+*Blake Prouty (that's me!)*, am from Sioux Falls, South Dakota  
+*Matt Doyle* lives in Sydney Australia  
+*Dicky Jiang* is based in Bandung Area, West Java, Indonesia
+
+**What skills do they bring?**
+
+*Kelvin Lockwood* - Product Ownership and Product Developer  
+*Blake Prouty (me again!)* - Full Stack Web Developer, HTML, CSS, JS, and React/Node  
+*Matt Doyle* - Full Stack Web Development: HTML, CSS, JS, and React/PHP/WP  
+*Dicky Jiang* - UIX Designer: Photoshop, Photography, Graphic Design, CSS, HTML, and Web Design
+
+**How was the experience of working remotely?**
+
+Could not have been better. Through the use of Github Projects, Google Documents, and Milanote we were able to work through open issues and tasks. Additionally, being that we were all in different time zones, someone was waking up as soon as the other was going to sleep, so we were nearly making progress around the clock.
+
+**How does it feel to be collaborating on a project at a time like this?**
+
+A great benefit, we’ve all shared with each other the additional steps we’ve taken to remain social and cope with the challenges we’ve faced. Whether it’s on a zoom-call or just in Slack, the frequent conversation and working towards a goal is therapeutic through and through. 
+
+**What tools, made available by the  hackathon, are you using?**
+
+At this time we’ve not had a need for a backend component on our project, but would like to thank [FaunaDB](https://fauna.com/) for offering their database services to the hackathon, I’ll be sure to use them in an upcoming project!
+
+**Is there anything else you’d like to say?**
+
+We would just like to thank everyone involved in this collaborative event and for allowing us to share what we’ve built!
+
+Blake *A.* Prouty
```

Comment:
Was the "A." accidentally italicized here?  Looks a bit weird:
>Blake *A.* Prouty

Reply:
I'll leave this for you to answer @johnny-newton. Seemed deliberate and looked kind of cool. I assumed it was a stylistic choice.

Reply:
We were both inside a google doc-him answering, me editing the Q+As for this post, and his icon read something like Amazing Armadillo or some such.  I asked him if he wanted to sign off as Mr. Armadillo, and opted to just drop an easter egg into his name instead. That's a dev thing right?

Reply:
I'm against in-jokes when the target audience is general public, but I won't die on this hill.